### DISCUSSION TITLE
I can't create a team on the pro plan

### DISCUSSION THREAD

**Discussion by escote-tatari** [vercel/community#5929] [Account Issue]
### Summary

![image](https://github.com/vercel/community/assets/75756504/025a45a6-beab-4b4a-beb4-92d48f9d29ad)

I am trying to setup an account for our contracting dev team to use. I created a team, input all of their email invites. Then tried to leave the team (because I don't want to pay $20 per month if I am not actively using the platform.  This deleted my team. When I tried to create a new one, complete with the CC info, it repeated gives this error.

It is very difficult to get started doing basic things on this platform.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/75756504/025a45a6-beab-4b4a-beb4-92d48f9d29ad)

I am trying to setup an account for our contracting dev team to use. I created a team, input all of their email invites. Then tried to leave the team (because I don't want to pay $20 per month if I am not actively using the platform.  This deleted my team. When I tried to create a new one, complete with the CC info, it repeated gives this error.

It is very difficult to get started doing basic things on this platform.

**Comment by escote-tatari** [comment]
wert

**Comment by amyegan** [comment]
Thanks for reporting this. We're investigating and will share updates here as new information becomes available 🙏

**Reply by amyegan** [reply]
I checked with the engineering team and they asked for some clarifying info to help them narrow down their investigation. 
- Are you still having this problem, or did you already find a solution?
- What are the names of the teams involved?

**Comment by amyegan** [comment]
Hey Scott! It looks like you were able to create a team and transfer ownership successfully. I'm going to close this discussion for now, but please reach out if you're still having trouble 🙏
